distributed_shampoo/utils/tests/shampoo_preconditioner_list_test.py

```diff
@@ -737,16 +760,25 @@ def test_update_preconditioners_and_precondition(self) -> None:
             R = G1^T * G1 + G2^T * G2 = [[1, 0], [0, 1]]
             P = L^{-1/4} G2 R^{-1/4} = 2^{-1/4} * [[0, 1]] = 2^{-1/4} G2
 
+        (4) Tensor of Size 0
+            G1 = 1
+            G2 = 2
+
+            L = [1] due to 0D tensor
+            P = L^{-1/2} G2 = 2
```

Comment:
Maybe it is safer to choose a different value for `G1`, e.g. 2, such that the inverse exponent actually matters for the result (`L != [1]`)?

Reply:
> I wonder does this should behave different mathematically on Shampoo?

I don't see a reason to not precondition gradients of scalar parameters, it should reduce to the same case as Adam for scalar parameters.